src/svelte/Components/Modals/EditMacro.svelte

```diff
@@ -8,17 +8,23 @@
   export let command;
   export let description;
   export let runable;
-  let error = false;
+  let nameError = false;
+  let commandError = false;
   let modal;
 
   function close() {
     modal.click();
-    error = false;
+    nameError = false;
+    commandError = false;
   }
 
   function submit() {
     if (name.trim() === "") {
-      error = true;
+      nameError = true;
+      return;
+    }
+    if (runable && command.trim() === "") {
```

Comment:
wo kommt denn jetzt das "runable" wieder her? :D

Reply:
*hatte in der letzten Story schon angemerkt, dass es "runnable" geschrieben wird